src/domain/auth-rebac/domain/src/repos/rebac_repository.rs

```diff
@@ -85,24 +85,15 @@ pub enum SetEntityPropertyError {
 #[derive(Error, Debug)]
 pub enum DeleteEntityPropertyError {
     #[error(transparent)]
-    EntityNotFound(EntityNotFoundError),
-
-    #[error(transparent)]
-    PropertyNotFound(EntityPropertyNotFoundError),
```

Comment:
I realized that for the implementation with databases, in one query we can't know exactly if there is no entity or no property because it's one row in a table.

Thus, the decision was made to merge the two errors into one.